.example.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_BENEFITSERVICE_BASE_URL=
+REACT_APP_KEYCLOAK_URL=
+REACT_APP_KEYCLOAK_REALM=
+REACT_APP_KEYCLOAK_CLIENT_ID=
+REACT_APP_CURAM_UA_LINK=
```

Comment:
missing 
NODE_ENV="development"
and probably maybe more

Reply:
I've included NODE_ENV and by default it is set to "development".
Are the quotations necessary? I'm not sure if they are meant to be included in the file